### PR TITLE
Dev -> Prod

### PR DESCRIPTION
**Pull request by jkonrath-postman**
TW-1196 Add collection template links to LC #5072

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/sending-requests/requests.md [hunk 1]
```diff
@@ -292,3 +292,5 @@ If your request doesn't work as expected, check out some [troubleshooting tips](
 ## Next steps
 
 Once you have your request set up, select __Send__ and examine the [Response](/docs/sending-requests/responses/).
+
+> <img alt="Collections icon" src="https://assets.postman.com/postman-docs/Collections.png#icon" width="24px"> To try out a collection template that has a typical REST API, select [REST API Basics](https://www.postman.com/templates/ddb19591-3097-41cf-82af-c84273e56719/REST-API-basics).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/sending-requests/requests.md:296]:
```suggestion
> <img alt="Collections icon" src="https://assets.postman.com/postman-docs/Collections.png#icon" width="24px"> To try out a collection template that has a typical REST API, select [REST API basics](https://www.postman.com/templates/ddb19591-3097-41cf-82af-c84273e56719/REST-API-basics).
```

Comment (MatthewPinsker-postman) [reply src/pages/docs/sending-requests/requests.md:296]: Really minor, but it looks like we're matching the casing used on the collection template overview page.

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

**Pull request merged**